arkane/ess/molpro.py

```diff
@@ -56,8 +57,56 @@ class MolproLog(ESSAdapter):
     MolproLog is an adapter for the abstract class ESSAdapter.
     """
 
-    def __init__(self, path):
+    def __init__(self, path, check_for_errors=True):
         self.path = path
+        if check_for_errors:
+            self.check_for_errors()
+
+    def check_for_errors(self):
+        """
+        Checks for common errors in a Molpro log file.
+        If any are found, this method will raise an error and crash.
+        """
```

Comment:
If possible, can you add a unit test for Molpro as well?

Reply:
I added a unit test by using the failed log files from ARC's unit tests